resources/cdk/README.md

```diff
@@ -84,3 +176,72 @@ This AWS CDK app creates the following resources:
   `username` partition key
 - An IAM role
   that enabless API Gateway to read from and write to the table.
+
+# Running a CDK app
+
+This section describes how to run any of these AWS CDK apps in this directory.
+ 
+Inputs (replace in code):
+- CLOUDFORMATION_TEMPLATE_NAME (For example, 'setup.yaml'.)
+- STACK_NAME (For example, 'myDynamoDBStack')
+
+ You can run a CDK app in several ways:
+
+ 1. To run this app with the AWS CDK, run the following command:
+ 
+    npm install && cdk synth && cdk deploy
+
+The names of the generated AWS resources will display in the output.
+
+To destroy the generated AWS resources after you are finished using them,run the following command:
```

Comment:
Note that in some cases, such as when an S3 bucket is not empty, the AWS CDK app cannot destroy a resource.